### DISCUSSION TITLE
Double quotes in replace

### DISCUSSION THREAD

**Discussion by Stetsyk** [EvgSkv/logica#189] [Q&A] [answered]
Hi @EvgSkv, 

I have SQL line:
`Replace(field_string1, '"', '') == field_string2`
How can I write this in logica?
There is no problem with other characters, only problems with replacing "

**Comment by EvgSkv** [comment, marked as answer]
Hi @Stetsyk ,
there is no string escaping and no single quotes just yet.
So quote symbol has to be specified via code. For example if using BiqQuery it's `SqlExpr("Chr(34)", {})` and if using SQLite it's `SqlExpr("Char(34)", {})`.
Example of using in context:
```
Text() = """
"The sun will rise", they said.
""";

ReplacedText() = Replace(Text(), SqlExpr("Chr(34)", {}), "");
```

Let me know if you have further questions and if it's a large hurdle for your usecase.

**Comment by Stetsyk** [comment]
Hi @EvgSkv,

Thank you for your answer. This is suitable for my case.
